styles.css

```diff
@@ -0,0 +1,138 @@
+.container {
+	background-image: url("https://backgrounddownload.com/wp-content/uploads/2018/09/background-banner-motor.jpg");
+	background-color:#000000;
+	text-align:center;
+	background-size: contain;
+	background-repeat: no-repeat;
+	overflow:auto;
+}
+
+.banner {
+	margin-left:170px;
+	float:left;
+	color:white;
```

Comment:
you have used hex codes previously so need to stick with one type of colour declaration

Reply:
also just set the text color to white on the body tag - that way, since all your text is white, you only need to do it once rather than everywhere you use it.